src/main/scala/org/mdedetrich/stripe/v1/Refunds.scala

```diff
@@ -278,11 +280,14 @@ object Refunds extends LazyLogging {
 
       val baseUrl = endpoint.url + s"/v1/refunds$totalCountUrl"
 
-      (baseUrl ?
-        ("charge"         -> refundListInput.charge) ?
-        ("ending_before"  -> refundListInput.endingBefore) ?
-        ("limit"          -> refundListInput.limit.map(_.toString)) ?
-        ("starting_after" -> refundListInput.startingAfter)).toString()
+      val query = Map(
+        "charge"         -> refundListInput.charge,
+        "ending_before"  -> refundListInput.endingBefore,
+        "limit"          -> refundListInput.limit.map(_.toString),
+        "starting_after" -> refundListInput.startingAfter
+      ).collect { case (k, Some(v)) => (k, v) }
```

Comment:
In order to not repeat yourself over and over, I've written `PostParams.flatten`. You should check it out.

If you find the name confusing, you're welcome to change it.